f2/src/FinalFeedbackPage.js

```diff
@@ -45,11 +45,10 @@ export const FinalFeedbackPage = () => {
               <Trans id="finalFeedback.title" />
             </H1>
             <FinalFeedbackForm
-              onSubmit={data => {
+              onSubmit={(data) => {
                 submitToServer(data)
                 setState((state.doneFinalFeedback = true))
-                if (state.formData && state.formData.submitted)
```

Comment:
Is this where the default values were causing the issue?

Reply:
the issue was that we were saving `submitted` in `state.formData`, and then the server would reject the submission as invalid (for having extra fields). The change here reflects that we have now moved `submitted` to `state`